src/common/app-paths/app-paths.test.ts

```diff
@@ -7,17 +7,17 @@ import getElectronAppPathInjectable from "../../main/app-paths/get-electron-app-
 import type { PathName } from "./app-path-names";
 import setElectronAppPathInjectable from "../../main/app-paths/set-electron-app-path/set-electron-app-path.injectable";
 import directoryForIntegrationTestingInjectable from "../../main/app-paths/directory-for-integration-testing/directory-for-integration-testing.injectable";
-import type { ApplicationBuilder } from "../../renderer/components/test-utils/get-application-builder";
-import { getApplicationBuilder } from "../../renderer/components/test-utils/get-application-builder";
+import type { ApplicationBuilder } from "../../features/test-utils/application-builder";
+import { setupInitializingApplicationBuilder } from "../../features/test-utils/application-builder";
 import type { DiContainer } from "@ogre-tools/injectable";
 import appPathsInjectable from "./app-paths.injectable";
 
 describe("app-paths", () => {
   let builder: ApplicationBuilder;
 
-  beforeEach(() => {
-    builder = getApplicationBuilder();
+  setupInitializingApplicationBuilder(b => builder = b);
```

Comment:
Ugh. This would be hiding a lot of important details. It also introduces dependency between order of these before each loops. If the only reason is to remove the requirement for `afterEach` from usages, then we should figure out why we need the `afterEach` at all. It shouldn't be there.